React/my-app/src/screens/login/index.js

```diff
@@ -28,18 +27,38 @@ class LoginContainer extends Component {
     if (
       this.validateData(event.target.email.value, event.target.password.value)
     ) {
-      this.setState({
-        emailError: null,
-        passwordError: null,
-        redirect: true
-      });
-      window.localStorage.signIn = true;
+      this.validateUser(event.target.email.value, event.target.password.value);
     }
   };
 
+  validateUser(email, password) {
+    postAccount(
+      "/users/sessions",
```

Comment:
use single quotes.

Reply:
mmm you shouldnt pass the endpoint by params.
In the service you should have a function to each api call, with his respective enpoint